### PR TITLE
delete all reserves dependency with the user

### PR DESCRIPTION
**Pull request by yvallejos**

### CODE REVIEW

File: app/components/users/users.service.js [hunk 1]
```diff
@@ -31,6 +33,16 @@
...
     }
+
+    function getReserves() {
+      var reserveRef = rootRef.child('reservation');
+      return $firebaseArray(reserveRef).$loaded();
+    }
+
+    function deleteReserve(reserveNumber){
+      var reserveRef = rootRef.child('reservation').child(reserveNumber);
+       return $firebaseObject(reserveRef).$remove();
```
Comment (Emmurillo) [inline app/components/users/users.service.js]: Indentation

File: app/components/reserves/delete/reserve.delete.controller.js [hunk 2]
```diff
@@ -11,7 +11,7 @@
     vm.deleteReserve = deleteReserve;
     vm.cancel = cancel;
 
-    function deleteReserve(reserve) {
+    function deleteReserve() {
       ReservesService.deleteReserve(reserve)
```
Comment (Emmurillo) [inline app/components/reserves/delete/reserve.delete.controller.js]: Where is this coming from now?

Comment (yvallejos) [inline app/components/reserves/delete/reserve.delete.controller.js]: @Emmurillo from the controller

Comment (johnnyxu-pernix) [inline app/components/reserves/delete/reserve.delete.controller.js]: @yvallejos I think what @Emmurillo meant was where is that value `reserve` coming from?

Comment (yvallejos) [inline app/components/reserves/delete/reserve.delete.controller.js]: @Emmurillo the controller gets the reserve id like parameter of the modal call

Comment (Emmurillo) [inline app/components/reserves/delete/reserve.delete.controller.js]: Sounds good

**Pull request merged**